frontend/src/hooks/useTeam.tsx

```diff
@@ -24,6 +24,18 @@ const useTeam = ({ autoFetchTeam = false }: AutoFetchProps): UseTeamType => {
 		}
 	});
 
+	const fetchTeam = useQuery(['team', teamId], () => getTeamRequest(teamId as string), {
```

Comment:
```suggestion
	const fetchTeam = useQuery(['team', teamId], () => getTeamRequest(teamId), {
```

You can avoid forcing the `teamId` as a string after setting the return type for `useTeamUtils`

Reply:
As the return type of router.query is: `string | string[] | undefined`, I have to keep the teamId as string, so that the function accepts the argument. Do you have another solution to avoid using the `as string` ? @nunocaseiro